app/javascript/packs/application.js

```diff
@@ -0,0 +1,18 @@
+/* eslint no-console:0 */
```

Comment:
all files in this directory should get removed...stick to your old custom one...

you ***did not need to run the webpack installer***

it was already there...